specification/servicefabricmesh/resource-manager/readme.md

```diff
@@ -0,0 +1,166 @@
+```
+### Tag: package-2018-07-01-preview
+
+These settings apply only when `--tag=package-2018-07-01-preview` is specified on the command line.
+
+
+``` yaml $(tag) == 'package-2018-07-01-preview'
+input-file:
+- Microsoft.ServiceFabricMesh/preview/2018-07-01-preview/servicefabricmesh.json
+```
+---
+# Code Generation
+
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_mgmt_service_fabric_mesh']
+```
+
+
+## C#
+These settings apply only when `--csharp` is specified on the command line. 
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.Azure.Management.ServiceFabricMesh
+  payload-flattening-threshold: 1
+  output-folder: $(csharp-sdks-folder)/ServiceFabric/Management.ServiceFabricMesh/Generated
+  clear-output-folder: true
+```
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.servicefabricmesh
+  package-name: azure-mgmt-servicefabricmesh
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-servicefabricmesh/azure/mgmt/servicefabricmesh
+```
+``` yaml $(python) && $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-servicefabricmesh
+```
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: servicefabricmesh
+  clear-output-folder: true
+```
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2018-07-01-preview
+```
+
+### Tag: package-2018-07-01-preview and go
+
+These settings apply only when `--tag=package-2018-02 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-02' && $(go)
+output-folder: $(go-sdk-folder)/services/servicefabricmesh/mgmt/2018-07-01-preview/servicefabricmesh

```

Comment:
Because this package is still in preview, please update this line to:

``` yaml
output-folder: $(go-sdk-folder)/services/preview/servicefabricmesh/mgmt/2018-07-01-preview/servicefabricmesh
```